app/controllers/sprints_controller.rb

```diff
@@ -9,18 +10,19 @@ class SprintsController < ApplicationController
   before_action only:[:update] {
     validate_sprint_date_parameter(params[:start_date], params[:end_date])
   }
-  before_action only: [:create, :update] {
-    validate_topics(params[:topic_ids], @course)
-  }
+  before_action :validate_topics, only: [:create, :update]
+  load_and_authorize_resource :except => [:create, :destroy, :create_collection, :update_collection]
 
-  # POST /courses/course_id/sprints
+  # POST /courses/:course_id/sprints
   def create
+    authorize! :create, Sprint, @course
     @sprint = @course.sprints.create!(sprint_params)
     json_response(@sprint, :created)
   end
 
-  # POST /courses/course_id/sprint-collection
+  # POST /courses/:course_id/sprint-collection
   def create_collection
+    authorize! :collection, Sprint, @course
```

Comment:
this imo doesnt have to be a distinct permission - anyone who can create sprints can create a collection, and everyone who can update sprints can update a collection